client/src/auth/index.tsx

```diff
@@ -36,10 +53,22 @@ export class AuthService {
     });
   }
 
+  getRoles() {
+    // REVIEW CHECK IF YOU ARE GOING TO NEED MORE THAN ONE ROLE
+    const [userRole] = this.user['https://poap.xyz/roles'];
+
+    return userRole;
+  }
+
   async handleCallback() {
     const result = await this.client.handleRedirectCallback();
     this._isAuthenticated = await this.client.isAuthenticated();
 
+    if (this._isAuthenticated) {
+      const _user = await this.client.getUser();
+      this.user = _user;
+    }
```

Comment:
Can we avoid the use of `_user`? since we're not using it anything else.

Code example:
```typescript
if (this._isAuthenticated) {
  this.user = await this.client.getUser();
}
```